pyseir/inference/infer_rt.py

```diff
@@ -324,10 +325,13 @@ def infer_all(self, plot=False, shift_deaths=0):
         df_all = None
         available_timeseries = []
 
-        if len(self.get_timeseries(TimeseriesType.NEW_CASES)) > 0:
+        cases = self.get_timeseries(TimeseriesType.NEW_CASES.value)[2]
+        deaths = self.get_timeseries(TimeseriesType.NEW_DEATHS.value)[2]
+
+        if np.sum(cases) > 5:
             available_timeseries.append(TimeseriesType.NEW_CASES)
 
-        if len(self.get_timeseries(TimeseriesType.NEW_DEATHS)) > 0:
+        if np.sum(deaths) > 5:
```

Comment:
Why 5?

Reply:
+1, would love come comments on what's going on here

Reply:
Promoted to configurable instance variables and added comment.